src/components/Badge/Badge.scss

```diff
@@ -19,12 +18,14 @@ $pip-spacing: ($height - $pip-size) / 2;
   @include pip-color(var(--p-icon, color('ink', 'lightest')));
   display: inline-flex;
   align-items: center;
+  min-width: $height;
```

Comment:
Prevent the badge from being taller than it is wide. In practice, this means it will be circular if the text inside is really narrow